components/popover/popover.vue

```diff
@@ -57,9 +58,8 @@
         }"
         :css="$attrs.css"
         :tabindex="contentTabindex"
-        @keydown.capture="onKeydown"
-        @after-leave="onLeaveTransitionComplete"
-        @after-enter="onEnterTransitionComplete"
+        v-bind="$attrs"
```

Comment:
Doing v-bind="$attrs" multiple times within the same component will duplicate classes in both elements, which might break styling or firing multiple events.

Reply:
Good catch, updated removing this and emiting the `keydown` event as it's treated as a custom event by defining in the emits https://github.com/dialpad/dialtone-vue/pull/570/commits/85363931daf5f1042ceaa1f8ca05401efc25d8bf